src/docs/get-started/test-drive/_androidstudio.md

```diff
@@ -9,10 +9,12 @@
  1. Click **Finish**.
  1. Wait for Android Studio to install the SDK and create the project.
 
-note: >-
-    <aside class="alert alert-info" role="alert" markdown="1">
-    <i class="fas fa-info-circle"></i> **Note:** When creating a new Flutter app, some Flutter IDE plugins ask for a company domain name in reverse order, something like `com.example`. The company domain name and project name are used together as the package name for Android (the Bundle ID for iOS) when the app is released. If you think the app might be released, it’s better to specify that now. The package name can’t be changed once the app is released, so make the name unique.
-
+{{site.alert.note}}
+When creating a new Flutter app, some Flutter IDE plugins ask for a company domain name in reverse order,
+something like `com.example`. The company domain name and project name are used together as the package name for Android
+(the Bundle ID for iOS) when the app is released. If you think the app might be released, it’s better to specify that now.
```

Comment:
Not sure why iOS's comment is in parentheses. 
"for Android (the Bundle ID for iOS)..." => "for Android and the Bundle ID for iOS...

"If you think the" => "If you think that the..."
(Rule:  https://developers.google.com/style/pronouns#optional-pronouns )

"it's better to specify that now" => "it's better to specify the package name now."
Also, I'd break this line on the comma and the previous line on the period.

Reply:
I put Bundle ID in parentheses because this section of the document Test drive is focused on Android Studio, so users will come across package name before Bundle Id; I also thought the parentheses broke up the sentence a little bit. I thought about reorganizing the sentence, too, by starting it with a dependent clause, but I didn't want to write two sentences with a dependent clause at the beginning. Anyways, I'm taking your suggestion. 

In my head I still heard the "that," but I can see why I didn't avoid ambiguity. Changed. 

Changed. 

Thanks, Shams for the comments.

Reply:
Good point about this being Android instructions. I remove my objection about the parenthetical.  :)